tests/platform_helper/fixtures/make_addons/expected/web/addons/terraform-my-s3-bucket.yml

```diff
@@ -0,0 +1,65 @@
+# Generated by platform-helper v0.1-TEST / 2023-08-22 16:00:00
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks:
+        - W2001 # Parameter not used
+
+Parameters:
+  # Copilot required Parameters...
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+
+Mappings:
+  myS3BucketEnvironmentConfigMap:
+    # Create an entry for each environment
+
+    development:
+      BucketName: 'my-bucket-dev'
+      KmsKeyArn: kms-key-not-found
+
+    production:
+      BucketName: 'my-bucket-prod'
+      KmsKeyArn: kms-key-not-found
+
+Resources:
+  myS3BucketS3AccessPolicy:
+    Metadata:
+      'aws:copilot:description': 'An IAM ManagedPolicy for your service to access the bucket'
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: Grants Read access to the S3 bucket.
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: KMSDecryptAndGenerate
+            Effect: Allow
+            Action:
+              - kms:Decrypt
+              - kms:GenerateDataKey
+            Resource: !FindInMap [myS3BucketitEnvironmentConfigMap, !Ref Env, KmsKeyArn]
+          - Sid: S3ObjectActions
+            Effect: Allow
+            Action:
+              - s3:GetObject
+
+            Resource: !Sub
+              - "arn:aws:s3:::${bucket_name}/*"
+              - bucket_name: !FindInMap [myS3BucketEnvironmentConfigMap, !Ref Env, BucketName]
+          - Sid: S3ListAction
+            Effect: Allow
+            Action: s3:ListBucket
+            Resource: !Sub
+              - "arn:aws:s3:::${bucket_name}"
+              - bucket_name: !FindInMap [myS3BucketEnvironmentConfigMap, !Ref Env, BucketName]
+Outputs:
+  myS3BucketAccessPolicy:
+    Description: "The IAM::ManagedPolicy to attach to the task role"
+    Value: !Ref myS3BucketS3AccessPolicy
```

Comment:
So much duplication in these three files. I wish we'd had time to do [DBTP-552 Reduce volume of test fixture files in `platform-tools`](https://uktrade.atlassian.net/browse/DBTP-552) already.

Reply:
Yep, agreed re's duplication. I don't think that's something we'd want to fix at this point though. Maybe later when the legacy code is removed.